src/components/sidebar/MarkerSidebar.js

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+
+import explicitConnect from '../../utils/connect';
+import { selectedThreadSelectors } from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import { formatMilliseconds } from '../../utils/format-numbers';
+import { CanSelectContent } from './index';
+
+import type {
+  ConnectedProps,
+  ExplicitConnectOptions,
+} from '../../utils/connect';
+import type { ThreadIndex } from '../../types/profile';
+import type {
+  IndexIntoTracingMarkers,
+  TracingMarker,
+} from '../../types/profile-derived';
+
+type SidebarDetailProps = {|
+  +label: string,
+  +value: React.Node,
+|};
+
+function SidebarDetail({ label, value }: SidebarDetailProps) {
+  return (
+    <React.Fragment>
+      <div className="sidebar-label">{label}:</div>
+      <div className="sidebar-value">{value}</div>
+    </React.Fragment>
+  );
+}
```

Comment:
I think `SidebarDetail` could also be extracted into its own file -- I think this is nice to make it easier to reuse in our various sidebars while still keeping a consistent style.

Reply:
Although if we use MarkerTooltipContents in the future we won't likely need SidebarDetail then.